### ISSUE TITLE
In sim2real task, the source and target are not describing the same scene which is meaningless for transfer learning

### ISSUE THREAD

**Issue by drowning-in-codes** [ucla-mobility/V2V4Real#35]
Hi, thank you for your excellent work. I noticed that source and target data were shuffled in the domain adaptation training.
The simulated opv2v and the real-world dataset v2v4 do not just have domain differences. For example, in the same frame, in opv2v, it could be three CAVs on a curvy road, while in v2v4, there could be one CAV on a straight road, so they have different labels.
```python
 source_train_loader = DataLoader(source_opencood_train_dataset,
                                     batch_size=hypes['train_params'][
                                         'batch_size'],
                                     num_workers=8,
                                     collate_fn=source_opencood_train_dataset.collate_batch_train,
                                     shuffle=True,
                                     pin_memory=False,
                                     drop_last=True)
  target_train_loader = DataLoader(target_opencood_train_dataset,
                                     batch_size=hypes['train_params'][
                                         'batch_size'],
                                     num_workers=8,
                                     collate_fn=source_opencood_train_dataset.collate_batch_train,
                                     shuffle=True,
                                     pin_memory=False,
                                     drop_last=True)
```
The domain adaptation configuration YAML file is as follows.
```yaml
name: corpbevtlidar_da
root_dir: '/data/opv2v/rain' # the path of the source domain
root_dir_target: '/data/v2v4real/train' # the path of the target domain
validate_dir: '/data/v2v4real/val'
```
I think it would make sense to combine the source data and target data into one scene with different point cloud distributions. 
Appreciate again.

**Comment by fengxueguiren** [comment]
Hi, you can refer to the setting of unsupervised domain adaptation.

**Comment by drowning-in-codes** [comment]
I may have thought of unsupervised domain adaptation too narrowly; I thought that domain adaptation differences would be similar between different kinds of dogs. However, the difference between the real environment and the simulated data may be too large (perhaps larger than between different types of dogs).
Anyway, OPV2V and V2V4Real are good work. Thanks again to the authors involved.
